app/models/post.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+class Post < ApplicationRecord
+  validates :title, uniqueness: true, presence: true
+  validates :body, presence: true
+  validates :author_id, presence: true
+
+  belongs_to :author
+  has_many :comment
```

Comment:
- The common convention for naming `has_many` association is plural `comments` instead of `comment` as it's more readable.

Reply:
fix